### DISCUSSION TITLE
Where do you start on Drupal?

### DISCUSSION THREAD

**Discussion by danielrosehill** [GetDKAN/dkan#4163] [Q&A]
Installed Drupal 10 and added DKAN as a module by downloading and uploading the Github repo.

That aspect worked fine but
.. to ask the very dumb newbie question... where so I go about configuring everything in the backend (The Drupal backend, that is)?

**Comment by janette** [comment]
@danielrosehill so you have a working Drupal site with DKAN enabled?
What specifically do you want to configure?

**Comment by dafeder** [comment]
@danielrosehill if you are able to log in to your Drupal site just go to /admin.
